packages/components/src/docs/General/10-FAQ.mdx

```diff
@@ -0,0 +1,71 @@
+
+### How do components, styles, samples, and patterns differ in our library?
+In our library, we offer four distinct types of elements:
+1. Components: These are lightweight [web components](https://developer.mozilla.org/en-US/docs/Web/API/Web_Components) constructed with lit-element.
+2. Styles: These are even lighter CSS classes that can be applied to any HTML element.
+3. Samples: These are practical demonstrations showcasing the usage of our components and styles within specific contexts.
+4. Patterns: These are predefined use-cases designed to address specific design requirements without the need for a dedicated component.
+Visit our [introduction page](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-introduction--docs#structure) for more details.
+
+### Which browsers are supported?
+TBA
+
+### How can I submit a feature request or report a bug?
+To do so, navigate to the issue section on GitHub and utilize our provided issue templates. Whether you're suggesting a new feature or reporting a bug, these templates offer placeholders and designated content sections where you can input the necessary information. Please ensure to include as many detailed specifics as possible when submitting your request or bug report.
+
+### Is there an endpoint to the development of the Solid Design System?
+No, the Solid Design System is an ongoing endeavor, much like the ever-evolving nature of the web itself. We are committed to continually refining and expanding its features, ensuring it remains aligned with the latest standards and technologies.
+
+### Is there a roadmap for upcoming features?
+Sure. You can explore our project board for a detailed overview of our [roadmap](https://github.com/orgs/solid-design-system/projects/1/views/48).
+
+### What we do and what we don't do?
+- ✅ Provide guidance and support with components, styles, samples and patterns
+- ✅ Create, update and maintain our components, styles, samples and patterns
+- ✅ Write documentation about the system’s elements’ behaviours and interactions
+- tba
+
+- ❌ Screen designs
+- ❌ Prototype flows
+- ❌ Specify your product's rules
+- ❌ Maintain business area libraries
+- tba
+
+### Does the use of the Solid Design System guarantee my website's accessibility?
+No, while our components, styles, samples, and patterns are crafted to fulfill WCAG 2.2 specifications independently, they do not ensure complete web accessibility on their own. Our offerings constitute only a portion of your website or application. Your usage, content, and overall structure play significant roles in achieving full accessibility. We aim to provide flexibility while encouraging you to consider these aspects to deliver a fully accessible experience.
+
+### Who is working on the Solid Design System?
+The Solid Design System team, consisting of Product Owner, Brand Designer, UX Designers, Developers and Architects ....
```

Comment:
Hm. For this question this feels quite unconcrete.

Reply:
The Solid Design System Team consists of Product Owner, Brand Designer, UX/UI Designers, Developers and Architects ....

Reply:
@mariohamann I was not sure about this one as well. If you think so, we could remove that one... This was a Q&A in most of the other systems as well, so I added it here. 

I see the point that providing some details about the team helps to "connect" but I personally don't want to be mentioned in person here...

Reply:
Do others answer it in the same way? But yeah, if you think it helps anyone – why not. 😃

Reply:
yes. thats basically the same others answer in this question. I think its more a way to show that we are an interdisciplinary team. I will add some more info here.